library/src/main/java/com/novoda/viewpageradapter/ViewPagerAdapter.java

```diff
@@ -51,7 +56,7 @@ public V instantiateItem(ViewGroup container, int position) {
      * @param position  the position of the data set that is to be represented by this view
      * @param viewState the state of the view
      */
-    protected void bindView(V view, int position, @Nullable SparseArray<Parcelable> viewState) {
+    private void bindView(V view, int position, @Nullable SparseArray<Parcelable> viewState) {
```

Comment:
@eduardb @ouchadam Is passing a `SparseArray` really the best way of accomplishing that? If I read through this code, there's nothing about this method that would make me think "okay, we need this `SparseArray` here to tell if we've bound to this view before or not". Wouldn't it be better to handle that on the view itself?